src/utils/SignalHandler.cpp

```diff
@@ -12,20 +11,10 @@ void terminationSignalHandler(int signal)
   // Print the events statistics
   precice::utils::EventRegistry::instance().signal_handler(signal);
 
-  // Delete stale .A-B.adress files
-  std::regex addrFile(R"(\..*-.*\.address)");
-
-  namespace fs = boost::filesystem;
-  fs::path cwd = fs::current_path();
-
-  for (fs::directory_iterator dir(cwd); dir != fs::directory_iterator(); ++dir) {
-    std::string file = fs::path(dir->path()).filename().string();
-    if (std::regex_match(file, addrFile)) {
-      fs::remove(dir->path());
-    }
-  }
-
-  std::exit(-1);
+  // Remove connection info files. This is just a guess and will only
+  // work if the address directory is ".".
+  boost::filesystem::remove_all(".precice");
```

Comment:
Isn't this a bit risky? It is probably a better idea to simply ignore everything.
It is quite simple to `rm` a single directory.

Reply:
We had it with the old connection files and it was added due to explicit user request. Problem is, that when you forget to `rm`, the connection will likely fail.